lib/api.ts

```diff
@@ -100,9 +100,14 @@ export function getContributorBySlug(file: string, detail = false) {
     ...activityData,
     activity: [...activityData.activity, ...getSlackMessages(data.slack)],
   };
+  let AllUnqPrReviews = new Set();
```

Comment:
Keeping this set as a global state could lead to inconsistent results. Prefer keeping it localized specific to a contributor.

Reply:
I don't think it causes inconsistent results, because for each contributor we are calculating separately.

and I didn't get how to keep it localized specific to a contributor.